.circleci/copy-junit-reports.js

```diff
@@ -19,18 +19,20 @@ const findTestResultsInDir = function (dirPath, files) {
     if (statSync(filePath).isDirectory()) {
       files = findTestResultsInDir(filePath, files);
     } else {
-      if (file === 'test.xml') {
+      // Only the test result files, which are XML with the .xml extension, should be discovered.
+      if (extname(file) === '.xml') {
         files.push(filePath);
       }
     }
   }
   return files;
 };
 
-/** StdOut result from the bazel info command. */
-const bazelInfoOutput = execSync('yarn bazel info', {stdio: 'pipe', encoding: 'utf8'});
 /** Absolute path to the bazel instance's testlog directory.  */
-const testLogPath = bazelInfoOutput.match(/bazel\-testlogs: (.*)/)[1];
+const testLogPath = execSync('yarn bazel info bazel-testlogs', {
```

Comment:
Nit: Should we use --silent for Yarn or will that pollute the stdout?

Reply:
Good call, I have it set to always be silent in my env, so I forget about it sometimes.